core/drivers/plic.c

```diff
@@ -282,3 +282,8 @@ void plic_it_handle(void)
 
 	plic_complete_interrupt(pd, id);
 }
+
+void interrupt_main_handler(void)
+{
+	plic_it_handle();
+}
```

Comment:
What if a specific platform that uses the PLIC driver needs to do something else?

Reply:
I referenced this commit https://github.com/OP-TEE/optee_os/commit/67e55c51c9149ea549664b3981ad9032dcf4ce7f.
But now I think you are right, we should not restrict platform's interrupt handling in this early stage.
I'll remove that commit.